pkg/kafka/auth/options.go

```diff
@@ -49,6 +49,14 @@ const (
 
 	defaultPlainTextUserName = ""
 	defaultPlainTextPassword = ""
+
+	// Scram configuration options
```

Comment:
Not a huge fan of the convention, but Go rules state that acronyms should be all in caps, so, `SCRAM` everywhere in the next few lines.